Makefile

```diff
@@ -58,7 +58,7 @@ build-slim:
 test: run-unit-tests
 
 .PHONY: lint
-lint:
+lint: build-slim build-test
```

Comment:
@invidian @iaguis this `build-test` added is also running some tests:

```
testing: warning: no tests to run
PASS
coverage: 10.0% of statements
ok  	github.com/kinvolk/lokomotive/pkg/components/calico-hostendpoint-controller	0.035s	coverage: 10.0% of statements [no tests to run]
?   	github.com/kinvolk/lokomotive/pkg/components/cert-manager	[no test files]
testing: warning: no tests to run
PASS
coverage: 6.5% of statements
ok  	github.com/kinvolk/lokomotive/pkg/components/cluster-autoscaler	0.062s	coverage: 6.5% of statements [no tests to run]
testing: warning: no tests to run
PASS
...
```

I guess this was not the intention?

Reply:
@rata are you sure it runs any tests? I don't think it should.

Reply:
Pasted the output when running `make lint` there... :)

Reply:
For each package, you get `testing: warning: no tests to run`, meaning tests does not run. We also run it with `-v` and it does not show any test title.

Example output from running tests:
```
?   	github.com/kinvolk/lokomotive/pkg/components	[no test files]
=== RUN   TestRenderManifest
--- PASS: TestRenderManifest (0.00s)
=== RUN   TestEmptyConfig
--- PASS: TestEmptyConfig (0.00s)
=== RUN   TestRenderManifestBadConfig
--- PASS: TestRenderManifestBadConfig (0.00s)
PASS
coverage: 70.0% of statements
```

Also `make build-test  | grep Test` does not print anything.

Why then you think `build-test` runs some tests?